apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -159,6 +160,112 @@ function CopyScheduleButton() {
+
+        return () => {
+            AppStore.off('customEventsChange', handleCustomEventsChange);
+            AppStore.off('currentScheduleIndexChange', handleCustomEventsChange);
+        };
+    }, []);
+
+    return (
+        <>
+            {customEvents.length > 0 && (
+                <Box>
+                    <Typography variant="h6">Custom Events</Typography>
+                    <Grid container spacing={1} padding={0}>
+                        {customEvents.map((customEvent) => {
+                            return (
+                                <Grid item md={12} xs={12} key={customEvent.title}>
+                                    <CustomEventDetailView
+                                        customEvent={customEvent}
+                                        scheduleNames={scheduleNames}
+                                        isSkeletonMode={isSkeletonMode}
+                                    />
+                                </Grid>
+                            );
+                        })}
+                    </Grid>
+                </Box>
+            )}
+        </>
+    );
+}
+
+interface ScheduleNoteBoxProps {
+    scheduleNote: string;
+    isSkeletonMode: boolean;
+}
+
+function ScheduleNoteBox(props: ScheduleNoteBoxProps) {
+    const { isSkeletonMode } = props;
+
+    const [scheduleNote, setScheduleNote] = useState(props.scheduleNote);
+    const [scheduleIndex, setScheduleIndex] = useState(AppStore.getCurrentScheduleIndex());
+
+    const handleNoteChange = useCallback(
+        (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+            setScheduleNote(event.target.value);
+            updateScheduleNote(event.target.value, scheduleIndex);
+        },
+        [scheduleIndex]
+    );
+
+    useEffect(() => {
+        const handleScheduleNoteChange = () => {
+            setScheduleNote(AppStore.getCurrentScheduleNote());
+        };
+
+        const handleScheduleIndexChange = () => {
+            setScheduleIndex(AppStore.getCurrentScheduleIndex());
+        };
+
+        AppStore.on('scheduleNotesChange', handleScheduleNoteChange);
+        AppStore.on('currentScheduleIndexChange', handleScheduleIndexChange);
+
+        return () => {
+            AppStore.off('scheduleNotesChange', handleScheduleNoteChange);
+            AppStore.off('currentScheduleIndexChange', handleScheduleIndexChange);
+        };
+    }, []);
+
+    return (
+        <Box>
+            <Typography variant="h6">Schedule Notes</Typography>
+
+            <TextField
+                type="text"
+                variant="filled"
+                label="Click here to start typing!"
+                onChange={handleNoteChange}
+                value={scheduleNote}
+                inputProps={{ maxLength: NOTE_MAX_LEN }}
+                InputProps={{ disableUnderline: true }}
+                fullWidth
+                multiline
+                disabled={isSkeletonMode}
```

Comment:
When the box is disabled, it's not clear in dark mode that it is disabled. One choice could be to add a `not-allowed` cursor CSS styling, but that would be inconsistent with the rest of the site.

An alternative choice would be to use a ternary in the label, with `"Editing is not allowed in skeleton mode"`.